cylc/flow/commands.py

```diff
@@ -448,9 +448,12 @@ async def force_trigger_tasks(
     flow: List[str],
     flow_wait: bool = False,
     flow_descr: Optional[str] = None,
+    on_resume: bool = False
 ):
     """Manual task trigger."""
     validate.is_tasks(tasks)
     validate.flow_opts(flow, flow_wait)
```

Comment:
```suggestion
    validate.flow_opts(flow, flow_wait)
    if on_resume:
        LOG.warning(
            "The --on-resume option is deprecated and will be removed "
            "at Cylc 8.5."
        )
```

Reply:
What are the bets this option is never used?!

Reply:
The point is, BOM need the option to be available *in case it is needed* to address a real problem that they have already encountered in the wild. (Broadcasting to past task that needs to be retriggered).  I would not have been proposing this option if we didn't have that curly one to deal with.